samples/qt/README.md

```diff
@@ -0,0 +1,99 @@
+
+# Astarte Device SDK C++ - Qt Example
+
+This example demonstrates how to use the **Astarte Device SDK for C++** in a **Qt** application.
+
+---
+
+## 📁 Example Location
+
+```
+astarte-device-sdk-cpp/samples/qt
+```
+
+---
+
+## 🛠️ Requirements
+
+- **Qt Creator**
+- **CMake**
+- **C++ compiler**
+- **gRPC and dependencies** (including protobuf)
+- **Astarte Device SDK C++** (fetched via CMake)
+
+---
+
+## ⚙️ Building the Example
```

Comment:
Those instructions are nice!
Could you also add a simple bash script that performs those actions without GUI?
So that this sample can also be built in CI.